core/modules/layout/layout.admin.inc

```diff
@@ -221,18 +221,18 @@ function layout_settings_form($form, &$form_state, Layout $layout) {
     '#required' => TRUE,
     '#access' => !$layout->isDefault(),
   );
-  $form['layout'] = array(
-    '#title' => t('Layout'),
+  $form['layout_template'] = array(
+    '#title' => t('Layout templates'),
```

Comment:
This should be `Layout template` as only one template can be selected for any given `Layout`.